src/RecoveryServices/RecoveryServices.SiteRecovery.Test/ScenarioTests/A2A/AsrA2ATests.ps1

```diff
@@ -664,8 +664,12 @@ function Test-VMNicConfig {
     #Update VM Nic properties
     $pe = Get-AzRecoveryServicesAsrReplicationProtectedItem -ProtectionContainer $pc -Name  $vmName
     $nicId = $pe.NicDetailsList[0].NicId
+    $ipConfigName = $pe.NicDetailsList[0].IpConfigs[0].Name
     $recNicName = getRecoveryNicName
-    $nicConfig = New-AzRecoveryServicesAsrVMNicConfig -NicId $nicId -ReplicationProtectedItem $pe -RecoveryNicName $recNicName -RecoveryNicResourceGroupName $recRgName -ReuseExistingNic
+
+    $ipConfig = New-AzRecoveryServicesAsrVMNicIPConfig -IpConfigName $ipConfigName -RecoverySubnetName ""
+    $ipConfigs = @($ipConfig)
+    $nicConfig = New-AzRecoveryServicesAsrVMNicConfig -NicId $nicId -ReplicationProtectedItem $pe -RecoveryVMNetworkId $RecoveryAzureNetworkId -RecoveryNicName $recNicName -RecoveryNicResourceGroupName $recRgName -ReuseExistingNic -IPConfig $ipConfigs
```

Comment:
just comma separated should work in argument. no need for creating an array

Reply:
leaving it for better readability as adding at cmdlet level in argument will make it more cumbersome